### PR TITLE
Multiple title support for DPUL

### PR DESCRIPTION
**Pull request by tpendragon**

### CODE REVIEW

File: app/services/manifest_builder.rb [hunk 1]
```diff
@@ -47,7 +47,7 @@ def initialize(resource)
     # Stringify the Resource by delegating to the header within the Decorator
     # @return [String]
     def to_s
-      decorate.header
+      resource.title.map(&:to_s)
```
Comment (hackmastera) [inline app/services/manifest_builder.rb]: looks like the yard docs are now out of sync with the method here.

Comment (tpendragon) [reply app/services/manifest_builder.rb]: Yup! Fixed.

**Comment by hackmastera** [comment]
When I looked at this not too long ago I didn't use the array because the iiif spec seemed to imply that label should be a string. Did I read the spec wrong? http://iiif.io/api/presentation/2.1/#label

**Comment by escowles** [comment]
@hackmastera I'm seeing a lot of "at least one label" type of language that implies multiple labels are OK.

**Comment by hackmastera** [comment]
Ah, great, thanks @escowles

**Pull request merged**